FBSnapshotTestCase/FBSnapshotTestController.m

```diff
@@ -181,6 +182,23 @@ - (BOOL)saveFailedReferenceImage:(UIImage *)referenceImage
                       identifier:(NSString *)identifier
                            error:(NSError **)errorPtr
 {
+    UIImage *diffImage = [referenceImage fb_diffWithImage:testImage];
+
+    [XCTContext runActivityNamed:identifier ?: NSStringFromSelector(selector) block:^(id<XCTActivity> _Nonnull activity) {
```

Comment:
It seems like the user should be able to turn this off or on. If you're running a truly massive amount of snapshot tests, you may not want to do this, as it might make Xcode unusable.

Reply:
Hmm that's a good point. I could easily inflate some of the tests we have locally to fail a couple hundred times if we want but perhaps using the Uber codebase would be a better litmus test?

Reply:
OK so I added a loop around a single test that was failing which ran 15000 times and caused 30000 failures which resulted in 90000 attachments.

I will admit that Xcode really chugged when the tests finished. It beach balled and was non-responsive for like 5-10 minutes while it processed all of the attachments. Navigating to test files as well as the test pane in the report navigator does take a bit of time to load but nothing that I would consider blocking.

If I change the loop to run 5000 times, causing 10000 failures and 30000 attachments there is nearly no issues at all.

Reply:
I created an issue to track adding the ability to turn this feature off https://github.com/uber/ios-snapshot-test-case/issues/89